src/Illuminate/Notifications/Events/BroadcastNotificationCreated.php

```diff
@@ -44,6 +51,10 @@ public function __construct($notifiable, $notification, $data)
         $this->data = $data;
         $this->notifiable = $notifiable;
         $this->notification = $notification;
+
+        if (method_exists($this->notification, 'broadcastNow')) {
+            $this->connection = $this->notification->broadcastNow() ? 'sync' : null;
```

Comment:
Why should we assume the sync connection is called "sync". We can name it whatever we want? It's only the driver which is "sync", not the connection.